service/controller/app/resource/clients/resource.go

```diff
@@ -112,7 +91,7 @@ func (r *Resource) addClientsToContext(ctx context.Context, cr v1alpha1.App) err
 		return nil
 	}
 
-	k8sClient, err := r.k8sClientCache.GetK8sClient(ctx, &cr.Spec.KubeConfig)
+	clients, err := r.k8sClientCache.GetClients(ctx, &cr.Spec.KubeConfig)
```

Comment:
Should the package now be clientcache instead of k8sclientcache?

Later when we remove the use of the controller context I think we can remove the clients resource and just have clientcache.

Reply:
Good point, yes we should use the correct name.